### PR TITLE
Adding docker & compose file for lawline dev

### PR DESCRIPTION
**Pull request by shree94mashal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,23 @@
-# docker-dev
\ No newline at end of file
+# docker develepment enviornment for lawline application
+
+#Lawline Dev enroviment setup on local ubnutu machine
+
+ - Checkout of git branch on your machine 
+    git clone https://github.com/petabloc/docker-dev.git
+
+ - Intsall leteast version of docker and docker-compose
```
Comment (dpeterka) [inline README.md]: Install latest

File: README.md [hunk 2]
```diff
@@ -1 +1,23 @@
...
\ No newline at end of file
+# docker develepment enviornment for lawline application
+
+#Lawline Dev enroviment setup on local ubnutu machine
+
+ - Checkout of git branch on your machine 
+    git clone https://github.com/petabloc/docker-dev.git
+
+ - Intsall leteast version of docker and docker-compose 
+   bash dockerInstllation.sh
```
Comment (dpeterka) [inline README.md]: dockerInstallation?

File: nginx.conf [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$uri - $is_args - args :::'
+                      '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
```
Comment (dpeterka) [inline nginx.conf]: only good if what's in front is passing X-FORWARDED-FOR is it?

File: README.md [hunk 4]
```diff
@@ -1 +1,23 @@
-# docker-dev
\ No newline at end of file
+# docker develepment enviornment for lawline application
+
+#Lawline Dev enroviment setup on local ubnutu machine
```
Comment (dpeterka) [inline README.md]: environment

File: README.md [hunk 5]
```diff
@@ -1 +1,23 @@
-# docker-dev
\ No newline at end of file
+# docker develepment enviornment for lawline application
```
Comment (dpeterka) [inline README.md]: development environment

File: README.md (hunk 1 above)
Comment (shree94mashal) [reply README.md]: will get added in next commit with all the other instructions with correction of some minor mistakes

File: README.md (hunk 2 above)
Comment (shree94mashal) [reply README.md]: Yes to install the latest version of docker on the ubuntu machine. It is just info for new people to docker

File: nginx.conf (hunk 3 above)
Comment (shree94mashal) [reply nginx.conf]: it is not required will remove that.

File: README.md (hunk 5 above)
Comment (shree94mashal) [reply README.md]: Committed all required changes. Can you please approve

**Pull request merged**